pystiche_papers/utils/misc.py

```diff
@@ -253,3 +255,14 @@ def load_state_dict_from_url(
         return cast(
             Dict[str, torch.Tensor], torch.load(cached_file, map_location=map_location)
         )
+
+
+T = TypeVar("T")
+
+
+def channel_progression(
```

Comment:
Could you add tests for this? If this is tested, you may also use this from other tests to make them more readable.